string[] commands_list
---
string[] plan_result
---
string[] command_status